context/discovery-ontology.template.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <title>Web of Things (WoT) Discovery Ontology</title>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c' class='remove'></script>
+    <script class='remove'>
+        var respecConfig = {
+            specStatus: "REC",
+            latestVersion: null,
+            editors: [{
+                name: "Andrea Cimmino"
+            }, {
+                name: "Farshid Tavakolizadeh"
+            }],
+            github: {
+                repoURL: "https://github.com/w3c/wot-discovery",
+                branch: "main"
+            },
+            shortName: "wot-discovery-ontology"
+        };
+    </script>
+</head>
+
+<body>
+    <section id='abstract'>
+        <p>
+            This document provides an ontology describing discovery mechanisms
+            for the Web of Things (WoT).
+            These mechanisms include Thing Description Directories (TDDs) as
+            well as Thing Links.
+        </p>
+    </section>
+    <section id='sotd'>
+        <p>
+            TODO
+        </p>
+    </section>
+    <section>
+        <h2>Introduction</h2>
+    </section>
+
+    <!-- axioms rendered from RDF definitions -->
+    %s
+
```

Comment:
<section>
        <h2>Change Log</h2>
            <p>
            The following lists the changes to the ontology for each version.  Semantic versioning is used, so major version changes are breaking,
minor version changes are substantial but non-breaking, and bug fix versions are small and non-breaking.
            </p>
            <dl>
            <dt>Version 1.0.1</dt><dd>Script used to render HTML from TTL.  TTL updated to include annotations and comments, and defining default namespace.
            </dd>
            <dt>Version 1.0.0</dt><dd>Initial release.  Placeholder HTML only.
            </dd>
            </dl>

Reply:
```suggestion
    <section id="changelog">
        <h2>Change Log</h2>
            <p>
            The following lists the changes to the ontology for each version.  Semantic versioning is used, so major version changes are breaking,
minor version changes are substantial but non-breaking, and bug fix versions are small and non-breaking.
            </p>
            <dl>
            <dt>Version 1.0.1</dt><dd>Script used to render HTML from TTL.  TTL updated to include annotations and comments, and defining default namespace.
            </dd>
            <dt>Version 1.0.0</dt><dd>Initial release.  Placeholder HTML only.
            </dd>
            </dl>
```